learn/experimental/metrics.md

```diff
@@ -0,0 +1,13 @@
+# Metrics
+
+The `/metrics` endpoint exposes data compatible with [Prometheus](https://prometheus.io/). You can monitor this endpoint to gain valuable insight into Meilisearch's behavior and performance. This information can then be used to optimize an application's performance, troubleshoot issues, or improve the overall reliability of a system.
+
+[Visit the GitHub discussion to learn how to use this feature](https://github.com/meilisearch/product/discussions/625).
+
+Are you using this feature? Meilisearch wants to hear from you! Tell us about your experience in the [GitHub discussion](https://github.com/meilisearch/product/discussions/625). All feedback is useful and helps make Meilisearch better and easier-to-use.
+
+::: warning
```

Comment:
Instead of a warning, should we add `learn/experimental/overview.md`? We could [reuse this one](https://github.com/meilisearch/documentation/pull/1514/files#diff-79a9d3ff91ffc3a2929e7906f4607d012227afe7c6d48ba419d812284d203423)

Reply:
Don't you think it's better to have all information in one place? Especially since we only have one experimental feature right now.

Rereading that page I also get the feeling it's a bit too long 🤔

Reply:
We can go with all information on one page